test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,38 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+
+
+
+    }
+
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
This commit shouldn't be part of this PR. In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the master branch that is in sync with the upstream master branch